assignments/lambda-classes.js

```diff
@@ -1 +1,201 @@
 // CODE here for your Lambda Classes
+
+class LambdaPerson{
+	constructor(attributes){
+		this.name = attributes.name;
+		this.age = attributes.age;
+		this.location = attributes.location;
+		this.gender = attributes.gender;
+	}
+	speak(){
+		console.log(`Hello my name is ${this.name}, I am from ${this.location}`);
+	}
+}
+
+class Instructor extends LambdaPerson{
+	constructor(attributes){
+  		super(attributes);
+  		this.specialty = attributes.specialty;
+		this.favLanguage = attributes.favLanguage;
+		this.catchPhrase = attributes.catchPhrase;
+	}
+	demo(subject){
+		console.log(`Today we are learning about ${subject}`);
+	}
+	grade(student, subject){
+		console.log(`${student.name} receives a perfect score on ${subject}`);
+	}
+}
+
+class Student extends LambdaPerson{
+	constructor(attributes){
+  		super(attributes);
+		this.previousBackground = attributes.previousBackground;
+		this.className = attributes.className;
+		this.favSubjects =  attributes.favSubjects;
+	}
+	listsSubjects(){
+		this.favSubjects.forEach(function(subject){
+			console.log(subject);
+		});
+	}
+	PRAssignment(subject){
+		console.log(`${this.name} has submitted a PR for ${subject}`);
+	}
+	sprintChallenge(subject){
+		console.log(`${this.name} has begun sprint challenge on ${subject}`);
+	}
+}
+
+class ProjectManager extends Instructor{
+	constructor(attributes){
+  		super(attributes);
+		this.gradClassName = attributes.gradClassName;
+		this.favInstructor = attributes.favInstructor;
+	}
+	standUp(channel){
+		console.log(`${this.name} announces to ${channel}, @${channel} standy times!​​​​​`);
```

Comment:
I think the `@${channel}` here was supposed to be `@channel` (like the slack `@channel` that pings everyone).